scripts/cherry-pick

```diff
@@ -197,9 +208,45 @@ main() {
 
     git commit -m "$msg" "${commit_files[@]}" ||
         fail "failed to commit '$msg'"
+    local_cpickhash=$(git show --quiet "--pretty=format:%H") ||
+        { error "failed git show most recent commit"; return 1; }
 
     git commit -m "update changelog" debian/changelog ||
         fail "failed to commit update to debian changelog."
+    drop_cherry_pick_from_daily_branch ${local_cpickhash} ${daily_remote}
+
+    return 0
+}
+
+main() {
+    local short_opts="hd:v"
+    local long_opts="help,daily-remote:,verbose"
+    local getopt_out=""
+    getopt_out=$(getopt --name "${0##*/}" \
+        --options "${short_opts}" --long "${long_opts}" -- "$@") &&
+        eval set -- "${getopt_out}" ||
+        { bad_Usage; return; }
+
+    local cur="" next="" DAILY_REMOTE="origin"
+
+    while [ $# -ne 0 ]; do
+        cur="$1"; next="$2";
+        case "$cur" in
+            -h|--help) Usage ; exit 0;;
+            -d|--daily-remote) DAILY_REMOTE=${next}; shift;;
+            -v|--verbose) VERBOSITY=$((${VERBOSITY}+1)); set -x;;
```

Comment:
Hrm, as much as I like set -x;  I don't like verbosity to just turn into tracing.  the existing cherry pick uses debug(); let's make use of debug [1|2|3] "my debug message" and then only bump VERBOSITY value.